src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -586,47 +575,35 @@ public float GetSaturation()
             return s;
         }
 
-        public int ToArgb()
-        {
-            return unchecked((int)Value);
-        }
+        public int ToArgb() => unchecked((int)Value);
 
-        public KnownColor ToKnownColor()
-        {
-            return (KnownColor)knownColor;
-        }
+        public KnownColor ToKnownColor() => (KnownColor)knownColor;
 
         public override string ToString()
         {
             if ((state & s_stateNameValid) != 0 || (state & s_stateKnownColorValid) != 0)
             {
-                return nameof(Color) + " [" + Name + "]";
+                return $"{nameof(Color)} [{Name}]";
             }
             else if ((state & s_stateValueMask) != 0)
             {
-                return nameof(Color) + " [A=" + A.ToString() + ", R=" + R.ToString() + ", G=" + G.ToString() + ", B=" + B.ToString() + "]";
+                return $"{nameof(Color)} [A={A}, R={R}, G={G}, B={B}]";
```

Comment:
How much do we care about the performance of `ToString()` for these types? While the string concat here previously did allocate an array, it may be cheaper than the `string.Format()`. Though, if this is mainly for debugging purposes, perhaps the readability improvement outweighs any perf efficiencies?